client/stylesheets/_toCat.scss

```diff
@@ -3544,6 +3561,44 @@ On click the images will be displayed at normal size to complete the effect
     margin-left: 4px;
 }
 
+.listingItemBrand, .listingItemColor, .listingItemSize, .listingItemCondition, .listingItemQuantity, .listingItemTrades {
+    // width: 50% !important;
+}
+
+.listingItemBrand a, .listingItemBrand span, .listingItemColor span, .listingItemSize span, 
```

Comment:
Selector `listingItemBrand` should be written in lowercase with hyphens<br>Selector `listingItemColor` should be written in lowercase with hyphens<br>Selector `listingItemSize` should be written in lowercase with hyphens<br>Selector `listingItemCondition` should be written in lowercase with hyphens<br>Selector `listingItemQuantity` should be written in lowercase with hyphens<br>Selector `listingItemTrades` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line<br>Line contains trailing whitespace